can/io/blf.py

```diff
@@ -236,6 +250,29 @@ def __iter__(self):
                                       data=can_data[:length],
                                       channel=channel - 1)
                         yield msg
+                    elif obj_type == CAN_FD_MESSAGE_64:
+                        (
```

Comment:
no braces needed here

Reply:
It's not required, however this is the preferred way to wrap long lines (according to PEP 8).

The "more" PEP 8 version would be:
```python
(
    channel,
    dlc,
    _,
    _,
    can_id,
    _,
    fd_flags,
) = CAN_FD_MSG_64_STRUCT.unpack_from(data, pos)[:7]
```

And matching the rest of the code in this module would be:
```python
(channel, dlc, _, _, can_id, _,
 fd_flags) = CAN_FD_MSG_64_STRUCT.unpack_from(data, pos)[:7]
```

I do not mind changing it to what you preferred.

Reply:
I actually did not know the first version was encouraged. Leave it as is or change it to the second one, you are right in that it is well readable.

Reply:
Interesting, I didn't know that. Then leave/change it as you prefer. (I'd choose the second one, but that's just me 😃)